projects/lkft-android/master.jinja2

```diff
@@ -0,0 +1,68 @@
+{% extends "testcases/android/master.jinja2" %}
+
+{% set timeout_common_tests_after_boot = timeout_common_tests_after_boot|default(20) %}
+
+{% set lava_job_name = "lkft-android-" + KERNEL_BRANCH + '-' + BUILD_NUMBER + '-' + job_name %}
```

Comment:
Looking at the default job_name
{% block job_name %}{{PROJECT_NAME}}-{{OS_INFO}}-{{test_name}}-{{KERNEL_BRANCH}}-{{BUILD_NUMBER}}{% endblock job_name %}

wouldn't it be sufficient to use this?
What do  you miss here?

Reply:
first I see it's defined in the testcases/master/template-master.jinja2 file, why it could not be defined in the root master.jinja2 file?

and another question, with the format of "{{PROJECT_NAME}}-{{OS_INFO}}-{{test_name}}-{{KERNEL_BRANCH}}-{{BUILD_NUMBER}}", when we need to search all the jobs for one build(BUILD_NUMBER is known), what should be input in the lava textbox? 

when OS_INFO is not defined, there will be "--" in the job name.

it's to override the job_name block, why use the {{test_name}} variable? test_name should be the name of test under the test actions, and there might be multiple tests defined in on job with different test_name.

Reply:
OS_INFO: thats correct, OS_INFO for lkft should be OE or Debian, for android I expect it to be android or should it be something else?

Format can be changed, we just have to come to an agreement what format make sense.

I have to look why its in template-master.jinja2 and not in the root master.jinja2

Reply:
OK, when it's defined in the root master.jinja2 file, and in the format of "{{PROJECT_NAME}}-{{OS_INFO}}-{{KERNEL_BRANCH}}-{{BUILD_NUMBER}}-{{job_name}}", the android definition here could be removed with the use of that in the root master.jinja2. I think that could be done in a separate change. Moved the {{job_name}} to the last place, that would help us to search all the jobs of the same build. like when we want to check all the jobs of the build#2532 of android13-5.10 branch db845c builds, we could input lkft-android-android13-5.10-2532 in the search box here:
https://lkft.validation.linaro.org/scheduler/device_type/dragonboard-845c?dt_length=25&dt_search=lkft-android-android13-5.10-2532&dt_page=1#dt_

Reply:
created a issue here: https://github.com/Linaro/lava-test-plans/issues/182